.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,6 +13,9 @@ assignees: ''
 ### PHP version
 <!-- Output of `php -v` -->
 
+### Tracer version
+<!-- Output of `php --ri ddtrace` -->
```

Comment:
Since this outputs a lot more info than just the version, would you be ok with this command instead?

```suggestion
<!-- Output of `php -r "echo phpversion('ddtrace').PHP_EOL;"` -->
```